### PR TITLE
EPMDJ-1759: Add data-test attributes

### PR DESCRIPTION
**Pull request by RomanAchilov**
Issue: EPMDJ-1759

### CODE REVIEW

File: src/pages/agent-full-page/coverage-plugin/overview/active-scope-actions/active-scope-actions.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ export const ActiveScopeActions = withRouter(
           <FinishScopeButton
             type="primary"
             onClick={() => dispatch(openModal('FinishScopeModal', scope))}
+            data-test="finish-scope-button"
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/overview/active-scope-actions/active-scope-actions.tsx]: use block:element:test-context notation

File: src/pages/agent-full-page/coverage-plugin/code-coverage-card/code-coverage-card.tsx [hunk 2]
```diff
@@ -27,24 +28,31 @@ export const CodeCoverageCard = codeCoverageCard(
...
-    </div>
-  ),
+    testContext,
+  }: Props) => {
+    return (
+      <div className={className}>
+        <Card header={header}>
+          <CardSection>
+            <TotalCoverage>
+              <Panel data-test={testContext}>
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/code-coverage-card/code-coverage-card.tsx]: use block:element:context notation

File: src/pages/agent-full-page/coverage-plugin/code-coverage-card/coverages-by-type/coverages-by-type.tsx [hunk 3]
```diff
@@ -48,14 +49,20 @@ export const CoveragesByType = coveragesByType(
               <CoverageItem key={testType}>
                 <TestTypeIcon type={testType} />
                 <TestTypeName>{testType.toLocaleLowerCase()}</TestTypeName>
-                <TestsCount>({testCount})</TestsCount>
+                <TestsCount
+                  data-test={`${testContext}:test-count-type:${testType.toLocaleLowerCase()}`}
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/code-coverage-card/coverages-by-type/coverages-by-type.tsx]: use block:element:context notation

Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/code-coverage-card/coverages-by-type/coverages-by-type.tsx]: locale?

File: src/pages/agent-full-page/coverage-plugin/coverage-plugin-header/action-section/action-section.tsx [hunk 4]
```diff
@@ -20,7 +20,12 @@ export const ActionSection = actionSection(
       <div className={className}>
         <Action>
           <ActionName>{label}</ActionName>
-          <Count onClick={onClick} type={count ? type : ''} clickable={Boolean(count)}>
+          <Count
+            onClick={onClick}
+            type={count ? type : ''}
+            clickable={Boolean(count)}
+            data-test={`count:${label}`}
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/coverage-plugin-header/action-section/action-section.tsx]: block?

File: src/pages/agent-full-page/coverage-plugin/project-methods-card/methods-section/methods-section.tsx [hunk 5]
```diff
@@ -26,23 +26,33 @@ export const MethodsSection = methodsSection(
...
+        >
           {totalCount}
         </Total>
         <AdditionalInfo>
           {!hideAdditionalInfo && (
             <>
               <AdditionalInfoItem>
                 <AdditionalInfoItemHeader>Covered</AdditionalInfoItemHeader>
-                <AdditionalInfoItemValue>{coveredCount}</AdditionalInfoItemValue>
+                <AdditionalInfoItemValue data-test={`${title.toLocaleLowerCase()}:covered`}>
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/project-methods-card/methods-section/methods-section.tsx]: locale?

File: src/pages/agent-full-page/coverage-plugin/project-methods-card/methods-section/methods-section.tsx [hunk 6]
```diff
@@ -26,23 +26,33 @@ export const MethodsSection = methodsSection(
 
     return (
       <div className={className}>
-        <Total onClick={() => totalCount && setIsModalOpen(true)} clickable={Boolean(totalCount)}>
+        <Total
+          data-test={`${title.toLocaleLowerCase()}:card`}
```
Comment (RomanAchilov) [inline src/pages/agent-full-page/coverage-plugin/project-methods-card/methods-section/methods-section.tsx]: locale?

Comment (RomanAchilov) [reply src/pages/agent-full-page/coverage-plugin/project-methods-card/methods-section/methods-section.tsx]: title = context

File: src/pages/agent-full-page/coverage-plugin/code-coverage-card/coverages-by-type/coverages-by-type.tsx [hunk 7]
```diff
@@ -48,14 +49,20 @@ export const CoveragesByType = coveragesByType(
               <CoverageItem key={testType}>
                 <TestTypeIcon type={testType} />
                 <TestTypeName>{testType.toLocaleLowerCase()}</TestTypeName>
-                <TestsCount>({testCount})</TestsCount>
+                <TestsCount
+                  data-test={`coverage-by-type:test-count:${testType.toLowerCase()}:${testContext}`}
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/code-coverage-card/coverages-by-type/coverages-by-type.tsx]:
```
data-test={`coverage-by-type:test-count:${testContext}-${testType.toLowerCase()}`}
```

File: src/pages/agent-full-page/coverage-plugin/code-coverage-card/coverages-by-type/coverages-by-type.tsx [hunk 8]
```diff
@@ -48,14 +49,20 @@ export const CoveragesByType = coveragesByType(
...
+                </TestsCount>
                 {showRecording && testTypes.includes(testType) && (
                   <RecordingWrapper>
                     <RecordingIcon />
                     <RecordingText>Rec</RecordingText>
                   </RecordingWrapper>
                 )}
-                <TestTypeCoverage>{`${percentFormatter(coverage)}%`}</TestTypeCoverage>
+                <TestTypeCoverage
+                  data-test={`coverage-by-type:test-type-coverage:${testType.toLowerCase()}:${testContext}`}
```
Comment (pavyarov) [inline src/pages/agent-full-page/coverage-plugin/code-coverage-card/coverages-by-type/coverages-by-type.tsx]: same

**Pull request merged**